gptty/__main__.py

```diff
@@ -18,6 +18,8 @@
 import asyncio
 import pandas as pd
 import nltk
+import socket
+from contextlib import closing
```

Comment:
Are we sure this is required? No need to change, but in the future, we should consider using a `with` statement to manage context instead, if possible.